package.json

```diff
@@ -22,6 +22,9 @@
   "engines": {
     "atom": ">=1.21.0 <2.0.0"
   },
+  "release": {
+    "branch": "next"
```

Comment:
This is the only change that I do not know its consequences.
Why was this added?

Reply:
hmm good point, the apm extension's documentation aren't 100% clear. I'm happy to remove it, but i think it expects a config in `package.json`. I would like to see if it works before tweaking this (worst case we get a @next release before removing it.)